### PR TITLE
Clean up Company model and Glassdoor Service by extracting out worker and parser

### PR DESCRIPTION
**Pull request by EmilyMB**
All green!

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -42,6 +44,16 @@
...
+    Sidekiq::Worker.clear_all
+  end
 end
 
 Capybara.javascript_driver = :poltergeist
+
+VCR.configure do |c|
+  c.cassette_library_dir = "fixtures/vcr_cassettes"
+  c.hook_into :webmock
+  allow_http_connections_when_no_cassette = true
```
Comment (EmilyMB) [inline spec/spec_helper.rb:58]: This is directly from the VCR docs.

File: app/workers/company_worker.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    if result
+      Company.find_by(name: company).update_attributes(
+        url: result["website"],
+        industry: result["industry"],
+        logo_url: result["squareLogo"],
+        overall_rating: result["overallRating"].to_f,
+        rating_description: result["ratingDescription"],
+        cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
+        seniorleadership_rating: result["seniorLeadershipRating"].to_f,
+        compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
```
Comment (EmilyMB) [inline app/workers/company_worker.rb:15]: If anyone has great ideas about how to make this cleaner, I'd love to hear about it.

Comment (nathanows) [inline app/workers/company_worker.rb:15]: Yeah, i think this is just one of those unavoidable ones, we're passing through a lot of data. Any style changes would probably just be other Hound violations, i think this is probably fine

**Comment by nathanows** [comment]
Looks like you updated the .gitignore, but I'm still seeing the vcr fixture file being pushed up here

**Comment by nathanows** [comment]
:space_invader: :spades: :spaghetti: :sparkle: :sparkler: :sparkles:

**Pull request merged**